src/main/java/com/minecolonies/coremod/colony/Colony.java

```diff
@@ -1514,14 +1516,14 @@ public void markCitizensDirty()
      *
      * @param achievement The achievement to trigger
      */
-    public void triggerAchievement(@NotNull final Achievement achievement)
+    public void triggerAchievement(@NotNull final MineColoniesAchievement achievement)
     {
         if (this.colonyAchievements.contains(achievement))
         {
             return;
         }
 
-        this.colonyAchievements.add(achievement);
+        //this.colonyAchievements.add(achievement);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3ACommentedOutCodeLine)